pycbc/inject/inject.py

```diff
@@ -229,11 +428,47 @@ def make_strain_from_inj_object(self, inj, delta_t, detector_name,
 
         return signal
         
-        
     def end_times(self):
         """Return the end times of all injections"""
         return [inj.get_time_geocent() for inj in self.table]      
+
+
+class InjectionSet(object):
+    """Manages sets of injections and injects them into time series.
     
+    Injections are read from either LIGOLW XML files or HDF files.
+
+    Parameters
+    ----------
+    sim_file : string
+        Path to an hdf file or a LIGOLW XML file that contains a
+        SimInspiralTable.
+    \**kwds :
+        The rest of the keyword arguments are passed to the waveform generation
+        function when generating injections.
+
+    Attributes
+    ----------
+    table
+    """
+
+    def __init__(self, sim_file, **kwds):
+        ext = os.path.basename(sim_file)
+        if ext.endswith(('.xml', '.xml.gz', '.xmlgz')):
+            self._injhandler = _XMLInjectionSet(sim_file, **kwds)
+        elif ext.endswith(('hdf', '.h5')):
```

Comment:
@cdcapano @ahnitz should this be ``ext.endswith(('.hdf', '.h5'))`` (note the missing ``.`` in front of ``hdf``)?